src/plugins/intel_cpu/src/nodes/mkldnn_normalize_node.cpp

```diff
@@ -859,8 +911,61 @@ bool MKLDNNNormalizeL2Node::isExecutable() const {
 
 void MKLDNNNormalizeL2Node::prepareParams() {
     const auto& dims = getParentEdgeAt(DATA)->getMemoryPtr()->getStaticDims();
-    setPostOps(kernel_attrs, dims, true);
-    execPtr = NormalizeL2Executor::getNormalizeL2Executor(attrs, kernel_attrs, dims);
+
+    // appendPostOps() also generate neccessary arguments for each op
```

Comment:
WHy cannot we just call setPostOps here? See InterpolateNode as an example.
Right now it looks like unnecessary code duplication.

Reply:
Yes, indeed, changed to call setPostOps(),  thanks!